packages.md

```diff
@@ -0,0 +1,10 @@
+# Packages
+
+This page contains packages recommended by HelloFresh Golang Clan. These are not just the cool repos around, but the packages that we found really practical and worth to use over the others.
+
+If you need more packages or just for inspiration, visit [Awesome Go](https://awesome-go.com/).
+
+## Database
```

Comment:
I think this goes beyond this PR and should be a separate PR or discussion. I would really like to see some short explanations "why use package `x`" on that page.